src/core/modules/StarWrapper.cpp

```diff
@@ -431,6 +429,7 @@ QString StarWrapper1::getInfoString(const StelCore *core, const InfoStringGroup&
 		if ((wdsObs>0) || (binary_sep>0.f))  // either have a WDS observation or a separation modelled by the binary orbit
 		{
 			// use separation and position angle from the binary orbit if available
+			qDebug() << "binary_sep: " << binary_sep << " binary_pa: " << binary_pa;
```

Comment:
Is this debug really need here?

Reply:
my bad, was checking someething else and forgot to remove the line